src/components/editor/page-properties.tsx

```diff
@@ -267,6 +305,72 @@ const PageProperties = ({ pageNumber, surveyId }: Props) => {
     });
   };
 
+  /**
+   * Handler for add new image button click
+   *
+   * @param e event
+   */
+  const handleAddNewImageClick = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    e.stopPropagation();
+    setFileUploadDialogOpen(true);
+  };
+
+  /**
+   * Upload image request to s3
+   *
+   * @param file file
+   */
+  const uploadImage = async (file: File) => {
+    try {
+      setUploadLoading(true);
+      const presignedUrl = await (
+        await fetch(config.imageUploadBaseUrl, {
+          method: "POST",
+          headers: {
+            authorization: `Bearer ${auth?.tokenRaw}`
+          },
+          body: JSON.stringify({
+            path: `images/${file.name}`,
+            contentType: file.type
+          })
+        })
+      ).json();
+
+      const formData = new FormData();
+      formData.append("file", file);
+      formData.append("Content-Type", file.type);
+
+      const uploadResponse = await fetch(presignedUrl.data, {
+        method: "PUT",
+        headers: {
+          "Content-Type": file.type
+        },
+        body: file
+      });
+
+      if (uploadResponse.status !== 200) {
+        throw new Error();
+      }
+      getBackgroundImages();
+      setUploadLoading(false);
+      setFileUploadDialogOpen(false);
+    } catch (e) {
+      console.error(strings.errorHandling.editSurveysScreen.imageNotUploaded, e);
```

Comment:
Is it possible to set the error into the snackbar?

Reply:
Added this to the setError handler used throughout the file. Otherwise I'm not sure what snack bar you are referring to